locales/en/common.json

```diff
@@ -7,6 +7,37 @@
   "form_invalid_alert": "Address form errors to proceed.",
   "go_to_kafka_instances": "Go to $t(common:kafka) Instances",
   "kafka": "Kafka",
+  "overview": {
```

Comment:
I'd move this to its own file. To do it just namespace the translations using : between the namespace and the key. Like:

`t('overview:alpha')`

Then run `npm run parse-i18n` to collect all the keys automatically.
A new overview.json file will be created, move the keys there (only those needed).

If a translation.json file gets created too, you have non-namespaced keys that have to be fixed.